src/components/Forms/QueueAddFormContainer.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { ReactFinalForm } from '@dhis2/ui'
+import history from '../../services/history'
+import { useSubmitQueue } from '../../hooks/queues'
+import QueueAddForm from './QueueAddForm'
+
+const { Form } = ReactFinalForm
+
+const QueueAddFormContainer = () => {
+    const redirect = () => {
+        history.push('/')
+    }
+    const [submitQueue] = useSubmitQueue({ onSuccess: redirect })
+
```

Comment:
```suggestion

const navigateHome = () => history.push('/')

const QueueAddFormContainer = () => {
    const [submitQueue] = useSubmitQueue({ onSuccess: navigateHome })

```

Very, very, very minor comment: `navigateHome` seems more descriptive to me (`onSuccess: navigateHome`). And, as `history` is being imported directly, the function can be outside of the render code.

Reply:
Yeah you're right, that's clearer. Updated it.